bundles/asdi/asdi-projection-change/component/card.js

```diff
@@ -21,6 +21,24 @@ function (view, callback) {
     this.errorListing = Oskari.clazz.create('Oskari.projection.change.view.ErrorListing');
     this.create(view);
 }, {
+    eventHandlers: {
+        AfterMapLayerRemoveEvent: function (event) {
+            this.checkUnsupportedLayers();
+        },
+        AfterMapLayerAddEvent: function (event) {
+            this.checkUnsupportedLayers();
```

Comment:
This results in a new click handler whenever a layer is removed from map -> leak